cpp/include/gandiva/gandiva_configuration.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2017-2018 Dremio Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GANDIVA_CONFIGURATION_H
+#define GANDIVA_CONFIGURATION_H
+
+namespace gandiva {
+
+/// \brief runtime config for gandiva
+///
+/// It contains elements to customize gandiva execution
+/// at run time.
+class GandivaConfiguration {
+  public:
+  GandivaConfiguration(const char byte_code_file_path[] = nullptr) {
+    byte_code_file_path_ = byte_code_file_path;
+  }
+
+  const char * get_byte_code_file_path() const {
```

Comment:
the google style spec recommends no prefix for get accessor. so, simply

byte_code_file_path()